src/broadcast/broadcastManager.ts

```diff
@@ -70,7 +69,7 @@ export class BroadcastManager {
     });
     this.dolphinConnection.on(ConnectionEvent.ERROR, (err) => {
       // Log the error messages we get from Dolphin
-      log.error("[Broadcast] Dolphin connection error\n", err);
+      this.emit(BroadcastEvent.LOG, "[Broadcast] Dolphin connection error\n", err);
```

Comment:
Again over here, this event should only emit a single string.  We're returning 2 things here, a string, and an error object.